src/main/java/FrameworksDrivers/DataExport.java

```diff
@@ -0,0 +1,34 @@
+package FrameworksDrivers;
+
+import UseCase.DatabaseManager;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class DataExport {
+
+    public void write_csv() throws IOException {
+        DataExportInterface dataExportInterface = DatabaseManager.getDatabaseManager();
+        Object[][] data_to_csv = dataExportInterface.writeDatabase();
```

Comment:
Perhaps you could explain the steps that you use here using Javadoc or comments since it is a bit hard to understand the line appending that you are doing in lines 21 and onward. The formatting of the raw data seems important (for manipulation purposes) yet a bit hard to understand at a glance.